app/controlplane/internal/service/workflowcontract.go

```diff
@@ -91,6 +94,15 @@ func (s *WorkflowContractService) Create(ctx context.Context, req *pb.WorkflowCo
 		return nil, err
 	}
 
+	token, err := usercontext.GetRawToken(ctx)
+	if err != nil {
+		return nil, status.Error(codes.Unauthenticated, "invalid token")
+	}
+
+	if err = s.contractUseCase.ValidateContractPolicies(req.RawContract, token); err != nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid policies")
```

Comment:
this probably hides the validation error, what if you use 

`handleUseCaseError` here instead?